java/daikon/Daikon.java

```diff
@@ -2291,8 +2296,9 @@ public static void setupEquality(PptTopLevel ppt) {
       //  named program points such as :::POINT (used by convertcsv.pl)
       //  will be treated as leaves.
       if (p.ppt_name.isCombinedExitPoint()
+          || p.ppt_name.isCombinedThrowPoint()
           || p.ppt_name.isEnterPoint()
-          || p.ppt_name.isThrowsPoint()
+          //        || p.ppt_name.isThrowsPoint()
```

Comment:
Please remove, rather than commenting out, code that is no longer correct.
If you leave code commented out, then explain why you are doing so, to help future readers of the code.